workshops/parallax_effect/README.md

```diff
@@ -0,0 +1,176 @@
+---
+name: 'Parallax Effect'
+description: 'Creating greater user experience on web using vanilla javascript'
+author: '@giridhar7632'
+---
+
+# Parallax Effect
```

Comment:
Could we have a more exciting title?

Reply:
^^^ following up

Reply:
I have no idea about the title
I am thinking about it. But no use
It's just an effect. I can't find any interesting title.😔

Reply:
Please let me know if you have anything in your mind

Reply:
I kinda like “parallax effect”, it sounds cool

Reply:
Okay